src/compiler/python_generator.cc

```diff
@@ -232,10 +232,13 @@ static void PrintAllComments(const DescriptorType* desc, Printer* printer) {
 bool PrintBetaServicer(const ServiceDescriptor* service,
                        Printer* out) {
   out->Print("\n\n");
-  out->Print("class Beta$Service$Servicer(object):\n", "Service",
-             service->name());
+  out->Print("class Beta$Service$Servicer(object):\n",
+   "Service", service->name());
   {
     IndentScope raii_class_indent(out);
+    out->Print("\"\"\"The Beta API is deprecated for 0.15.0 and later. It is recommended\n"
+    "to use the GA API for all further purposes. This function was generated only\n"
+    "to ease transition from grpcio<0.15.0 to grpcio>=0.15.0\"\"\"\n");
     PrintAllComments(service, out);
```

Comment:
Should this be removed? I feel like this should be removed. How does the generated code element look both with it and without it?